.github/workflows/e2e-testing.yaml

```diff
@@ -26,6 +26,13 @@ jobs:
     if: ${{ needs.change-detection.outputs.project == 'true' }}
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+          cache: 'pip'
+      - run: python -m pip install awscli
```

Comment:
have we considered using the GH action instead https://github.com/marketplace/actions/aws-cli

Reply:
I will confirm that, probably don't need this. I had to add this when testing actions locally.

Reply:
updated Github actions come with pre-installed AWS CLI, so this isn't required